app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsController.kt

```diff
@@ -1,5 +1,6 @@
 package org.simple.clinic.search.results
```

Comment:
Can you change this commit message to say what you are doing instead of what you are fixing?
Eg:  `Use custom ViewHolder for all types in PatientSearchResultsItemType`

Reply:
done